chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -150,7 +150,10 @@ def self.unclaimed(node, include_mounted=false)
               %x{lsblk #{d.name.gsub(/!/, "/")} --noheadings --output MOUNTPOINT | grep -q -v ^$}
               next if $?.exitstatus == 0
             end
-            d.fixed and not d.claimed?
+            # skip claimed disks and multipath holders
```

Comment:
yeah, Im struggling to find the correct words here as to not make it confusing when someone reads it in the future. devices held by holders seems like also something that could lead to confusing from someone reading it for the first time. Me no good with big words :/